hazelcast/src/test/java/com/hazelcast/client/partitionservice/ClientPartitionLostListenerTest.java

```diff
@@ -70,6 +71,15 @@ public void tearDown() {
         hazelcastFactory.terminateAll();
     }
 
+    @Test
+    public void test_partitionLostListener_registeredByConfig() {
```

Comment:
Would be good to have a test similar to `test_partitionLostListener_invoked` just to make sure listeners are really invoked.